vectorizers/_vectorizers.py

```diff
@@ -422,16 +422,16 @@ def preprocess_tree_sequences(
             result_sequence.append((result_matrix, label_sequence))
     else:
         result_sequence = []
-        if MASK_STRING in token_dictionary:
-            _ = token_dictionary.pop(MASK_STRING)
+        if masking in token_dictionary:
+            _ = token_dictionary.pop(masking)
 
         for adj_matrix, label_sequence in tree_sequences:
             new_labels = [
-                label if label in token_dictionary.keys() else MASK_STRING
+                label if label in token_dictionary.keys() else masking
```

Comment:
```suggestion
                label if label in token_dictionary else masking
```
I don't believe we actually need the keys statement here.